indico/modules/events/timetable/controllers/manage.py

```diff
@@ -47,17 +47,9 @@ def _process(self):
                                                  timetable_data=timetable_data, timetable_layout=self.layout)
 
 
-class RHTimetableREST(RHManageTimetableBase):
+class RHTimetableREST(RHManageTimetableEntryBase):
```

Comment:
Well we need the logic to get the entry if an entry id is present.. I agree that the naming is not perfect